lib/flatfile/csvActorExport.class.php

```diff
@@ -57,28 +57,27 @@ private function exportRelations($filename, $resource)
   {
     $rows = array();
 
-    foreach ($resource->getActorRelations() as $item)
+    foreach ($resource->getActorRelations() as $relation)
     {
-      $relatedEntity = $item->getOpposedObject($resource->id);
+      $relatedEntity = $relation->getOpposedObject($resource->id);
 
-      if (QubitTerm::ROOT_ID == $item->type->parentId)
+      // Determine appropriate relation type term
+      $relationType = $relation->type;
+      if ($relation->subjectId == $resource->id)
       {
-        $category = $item->type;
-      }
-      else
-      {
-        $category = $item->type->parent;
+        $converseRelation = $relationType->getConverseActorRelationTerm();
```

Comment:
Nitpick: I think it would be helpful to add a comment here about why you want the converse relation term, if there is one.  I assume it's because the "forward" direction for the relation type is information_object -> actor, and this is going the other direction?

Reply:
That's the reason, yeah... I'll add a comment to clarify that.

Reply:
(Added. Also renamed a function and added another comment to make export functionality clearer.)

Reply:
Wow, great comment! 👍

Reply:
Thanks mister! :D